test/integ/test_easy_logging.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+import os
+from logging import getLogger
+from pathlib import Path
+from test.integ.conftest import create_connection
+
+import pytest
+
+pytestmark = pytest.mark.skipolddriver
+try:
+    import tomlkit
+
+    from snowflake.connector.config_manager import CONFIG_MANAGER
+    from snowflake.connector.constants import CONFIG_FILE
+except ModuleNotFoundError:
+    pass
```

Comment:
Is it really best to continue on if these imports fail?  Won't you get name errors in some of the tests if you do?  Also, should the `snowflake.connector` imports actually ever fail given that this is testing that package?

Reply:
This approach is to pass the old driver test in merge gate, that merge gate will test against a old python connector which does not have this packages

Reply:
Okay, I don't understand why this is necessary.  I'll start a separate conversation with you and Mark